js/ui.js

```diff
@@ -0,0 +1,47 @@
+"use strict"
```

Comment:
Start of the new UI class for some reusable graphical functions like buttons that are made consistent by these font and color settings at the top.